LibTest/io/HttpClient/findProxy_A01_t01.dart

```diff
@@ -51,12 +52,17 @@ test() async {
           .set(HttpHeaders.proxyAuthenticateHeader, 'Basic, realm=realm');
       request.response.statusCode = HttpStatus.proxyAuthenticationRequired;
       request.response.close();
+      // Shutdown server. We are not expecting any additional requests in case
+      // of DIRECT connection
+      Future.delayed(Duration(milliseconds: 100)).then((_) {
```

Comment:
Embedding assumptions about timings seems to be unfortunate - they can lead to flaky results when run on systems under heavy load for example.
Is there a way to change the test to get rid of `Future.delayed`?
What if client sends two requests, can server then assert that by the time second request arrives findProxy/authenticaeProxy should definitely be called/not called?

Reply:
@aam thank you. Tests rewritten to not to use hardcoded delays. Please rerewiew